affiliatewp-order-details-for-affiliates.php

```diff
@@ -142,26 +169,26 @@ private function includes() {
 	}
 
 	/**
-	 * Setup the default hooks and actions
+	 * Setup the default hooks and actions.
```

Comment:
```suggestion
	 * Sets up the default hooks and actions.
```
Third-person singular verb in a DocBlock summary.